docs/integrations/sources/postgres.md

```diff
@@ -168,6 +168,7 @@ Airbyte uses [logical replication](https://www.postgresql.org/docs/10/logical-re
 - Schema changes are not supported automatically for CDC sources. Reset and resync data if you make a schema change.
 - The records produced by `DELETE` statements only contain primary keys. All other data fields are unset.
 - Log-based replication only works for master instances of Postgres.  CDC cannot be run from a read-replica of your primary database.
+- Am Airbyte database source using CDC replication can only be used with a single Airbyte destination.  
```

Comment:
Could you add more context on why this is the case?

Reply:
👍 a little more info added.